orka-ipsw-setup.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+
+ORKA_VM_TOOLS_VERSION="${ORKA_VM_TOOLS_VERSION:-3.5.0}"
+
+GREEN="\033[0;32m"
+YELLOW="\033[1;33m"
+RED="\033[0;31m"
+NC="\033[0m"
+
+set -euo pipefail
+
+log() {
+    echo -e "${GREEN}[INFO]${NC} $1"
+}
+
+warn() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+error() {
+    echo -e "${RED}[ERROR]${NC} $1" >&2
+    exit 1
+}
+
+# Check if running as root
+if [[ $EUID -eq 0 ]]; then
+    error "This script should not be run as root. Please run as a regular user with sudo privileges."
+fi
+
+# Check if sudo is available
+if ! command -v sudo &> /dev/null; then
+    error "sudo is required but not available"
+fi
+
+log "Starting MacOS Orka VM setup..."
+
+# Enable Screen Sharing
+enable_screen_sharing() {
+    log "Enabling Screen Sharing..."
+    
+    # Enable Screen Sharing via System Preferences
+    sudo /System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart \
```

Comment:
Now we are enabling Remote Management (instead of just screen sharing). Is this what we want?

Reply:
updated this in the most recent commit